presto-product-tests/src/main/java/io/prestosql/tests/hive/TestSqlStandardAccessControlChecks.java

```diff
@@ -212,6 +212,15 @@ public void testAccessControlSelectWithCaseSensitiveUserName()
         assertThat(caseSensitiveUserNameExecutor.executeQuery(format("SELECT * FROM %s", viewName))).hasNoRows();
     }
 
+    @Test(groups = {AUTHORIZATION, PROFILE_SPECIFIC_TESTS})
+    public void testAccessControlSetTableAuthorization()
+    {
+        assertThat(() -> bobExecutor.executeQuery(format("ALTER TABLE %s SET AUTHORIZATION bob", tableName)))
```

Comment:
I'm not sure I understand the case of this test. Maybe a better name would make it clearer? What kind of scenario is it?

Reply:
It's a simple smoke test of whether only an owner can set table authorization.
It checks the `checkCanSetTableAuthorization` method in SqlStandardAccessControl.
Again this is compliant with the rest of this class as of naming.